lib/src/coap_media_type.dart

```diff
@@ -81,54 +88,119 @@ class CoapMediaType {
 
-  /// application/exi
-  static const int applicationExi = 47;
+  /// application/cwt
+  static const int applicationCwt = 61;
 
-  /// application/fastinfoset
-  static const int applicationFastinfoset = 48;
+  /// application/multipart-core
+  static const int applicationMultipartCore = 62;
 
-  /// application/soap+fastinfoset
-  static const int applicationSoapFastinfoset = 49;
+  /// application/cbor-seq
+  static const int applicationCborSeq = 63;
 
-  /// application/json
-  static const int applicationJson = 50;
+  /// application/cose-key
+  static const int applicationCoseKey = 101;
+
+  /// application/cose-key-set
+  static const int applicationCoseKeySet = 102;
+
+  /// application/senml+json
+  static const int applicationSenmlJson = 110;
+
+  /// application/sensml+json
+  static const int applicationSensmlJson = 111;
 
-  /// application/x-obix-binary
-  static const int applicationXObixBinary = 51;
+  /// application/senml+cbor
+  static const int applicationSenmlCbor = 112;
+
+  /// application/sensml+cbor
+  static const int applicationSensmlCbor = 113;
+
+  /// application/senml-exi
+  static const int applicationSenmlExi = 114;
+
+  /// application/sensml-exi
+  static const int applicationSensmlExi = 115;
+
+  /// application/coap-group+json
+  static const int applicationCoapGroupJson = 256;
+
+  /// application/dots+cbor
+  static const int applicationDotsCbor = 271;
+
+  /// application/missing-blocks+cbor-seq
+  static const int applicationMissingBlocksCborSeq = 272;
+
+  /// application/pkcs8
+  static const int applicationPkcs8 = 284;
+
+  /// application/csrattrs
+  static const int applicationCsrattrs = 285;
+
+  /// application/pkcs10
+  static const int applicationPkcs10 = 286;
+
+  /// application/pkix-cert
+  static const int applicationPkixCert = 287;
+
+  /// application/senml+xml
+  static const int applicationSenmlXml = 310;
+
+  /// application/sensml+xml
+  static const int applicationSensmlXml = 311;
+
+  /// application/senml-etch+json
+  static const int applicationSenmlEtchJson = 320;
+
+  /// application/senml-etch+cbor
+  static const int applicationSenmlEtchCbor = 322;
+
+  /// application/td+json
+  static const int applicationTdJson = 432;
+
+  /// application/vnd.ocf+cbor
+  static const int applicationVndOcfCbor = 10000;
+
+  /// application/oscore
+  static const int applicationOscore = 10001;
+
+  /// application/javascript
+  static const int applicationJavascript = 10002;
+
+  /// text/css
+  static const int textCss = 20000;
+
+  /// image/svg+xml
+  static const int imageSvgXml = 30000;
 
   /// any
   static const int any = 0xFF;
 
   /// Checks whether the given media type is a type of image.
   /// True iff the media type is a type of image.
   static bool isImage(int mediaType) =>
-      mediaType >= imageGif && mediaType <= imageTiff;
+      mediaType >= imageGif && mediaType <= imagePng;
 
   /// Is the media type printable
   static bool isPrintable(int? mediaType) {
```

Comment:
I guess the new JSON and XML based formats could probably be added here, right? (Also `textCss` and `applicationJavascript` I guess?) Are there any other criteria for a media type to be printable?

Reply:
YEp sounds OK, I guess the only criteria is that the output is readable when printed.